deploy/bring-your-own-ai-model.md

```diff
@@ -0,0 +1,69 @@
+## Using AI-Models
+
+There are three ways to load an AI Model with this pipeline. One can use one of the three methods listed below.
+
+### Use the publicly hosted Custom Vision Model as GitHub Packages. 
```

Comment:
I think this needs to be generic and not mention Custom Vision Model. The three methods that we documented in the earlier version were to illustrate the extensibility for hosting Custom Vision model.

Also, if our goal is to simplify the documentation, we should only point to the approach where they upload their AI Model container image to the ACR deployed by this solution. 

@mandarinamdar  Thoughts?

Reply:
I think the file name is incorrect. We should be calling the file as using-ai-model-in-pipeline. We are not elaborating here about BYO AI model, that is covered in this doc. But mainly that how to use AI models in a pipeline

Reply:
I think based on our discussion last week we decided to create issue #88 to cover the instructions on bringing new AI models.